src/sinks/util/udp.rs

```diff
@@ -287,10 +287,9 @@ where
             while let Some(mut event) = input.next().await {
                 let byte_size = event.size_of();
 
-                self.acker.ack(1);
-
                 self.transformer.transform(&mut event);
 
+                let finalizers = event.take_finalizers();
                 let mut bytes = BytesMut::new();
                 if encoder.encode(event, &mut bytes).is_err() {
                     continue;
```

Comment:
I think you'll need to update the acker here…

Reply:
So this didn't happen in your `console` commit that did just the deferral, but shouldn't we also drop the finalizers as we're acking here - later on the status updates happen in these stops, but if we hit these paths we'll also end up skipping the single `drop` that happens further down in the loop.